helix-front/src/app/cluster/cluster-list/cluster-list.component.ts

```diff
@@ -29,7 +28,6 @@ export class ClusterListComponent implements OnInit {
 
   ngOnInit() {
     this.loadClusters();
-    this.clusterService.can().subscribe((data) => (this.can = data));
```

Comment:
This change removes the broken permissions check from cluster creation.

Reply:
Here is where this `.can()` permissions method is defined: 

https://github.com/apache/helix/blob/master/helix-front/src/app/core/helix.service.ts#L14

Reply:
I'm looking at this again and seeing if there is a way to fix the bug while preserving the permission check. 🤔

Reply:
Found it! A better solution is to call the login check method again after the clusters load.